client/app/components/Modal/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import styles from "./styles.scss";
+import bulma from 'bulma';
+import avatar from "./IDICIA-logo-clear-center.svg";
+
+class ContactModal extends React.Component {
```

Comment:
look at our other stateful components for how we're doing this, we're using anon functions assigned to a const. 
@stern-shawn thoughts on state?

Reply:
Something like functional setState would just be a nice-to-learn instead of necessary at this point. The old syntax still works fine for simple things like toggles in this use case.